src/Microsoft.AspNet.Mvc.Core/ViewComponents/ViewComponentDescriptor.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Mvc
```

Comment:
I considered it. It shows up in a few places the user might touch, like `ViewComponentContext`. The other classes are generally only for extensibility so it makes sense.

Let's try it and see if it's horrid.